dag-store.js

```diff
@@ -4,13 +4,10 @@ import uuid from 'node-uuid';
 let nodes = (state = [], action = {}) => {
   switch(action.type) {
```

Comment:
Hey I am so sorry I did not get to this.  Craziness at work :( I will implement this change.

Thanks

Jim